src/index.html

```diff
@@ -11,6 +11,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="forms-wrapper">
+      <form
+        action="/"
+        method="get"
+        data-qa="big"
+        class="form"
+      >
+        <label for="big-search" class="label-search-icon">
+          <img
+            class="icon"
+            src="images/Search.svg" alt="Search icon">
+        </label>
+
+        <input
+          id="big-search"
+          class="search"
+          type="search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          name="bigSearchText"
+        >
+      </form>
+
+      <form
+        action="/"
+        method="get"
+        data-qa="small"
+        class="form"
+        >
```

Comment:
Also here the indent is incorrect, look on the indent of your inputs or img below